plugins/modules/clustering/nomad/nomad_job.py

```diff
@@ -0,0 +1,249 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of nomad server.
+      required: true
+      type: str
+    secure:
+      description:
+        - TLS/SSL connection.
+      type: bool
+      default: false
+    timeout:
+      description:
+        - Timeout (in seconds) for the request to nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - Enable TLS/SSL certificate validation.
+      type: bool
+      default: true
+    cert:
```

Comment:
This is about client certificates, right? In that case, the name should be `client_cert`.

Reply:
ok